handbook/bizops/index.md

```diff
@@ -29,12 +29,20 @@ Here are the following sources we collect data from:
 * Sourcegraph.com Site-admin pages: customer subscriptions and license keys
 * [Pings](https://docs.sourcegraph.com/admin/pings) from self-hosted Sourcegraph instances containing anonymous and aggregated information
 * [Custom tool to track events](https://github.com/sourcegraph/sourcegraph/issues/5486) on the Sourcegraph.com instance
+* [Prometheus search data](https://sourcegraph.com/-/debug/grafana/d/0ZBSEmRWk/searcher?orgId=1&refresh=5s) represents the breakdown of search statuses (success, error, timeout, partial timeout) on [Sourcegraph.com](https://sourcegraph.com/search)
```

Comment:
How did you pick this one? If I'm not mistaken, there are a number of dashboards — if you haven't already, Uwe or someone else on the distribution team may be able to help find useful ones.

Reply:
Good call, that was the one we were specifically using for an OKR but I'll share the folder of dashboards instead